shared/src/e2e/screenshotReporter.ts

```diff
@@ -1,54 +1,45 @@
 import mkdirp from 'mkdirp-promise'
 import * as path from 'path'
-import puppeteer from 'puppeteer'
+import * as puppeteer from 'puppeteer'
+import { afterEach } from 'mocha'
 
 /**
  * Registers a jasmine reporter (for use with jest) that takes a screenshot of the browser when a test fails (and
```

Comment:
Not entirely sure, but this caught my eye:

```suggestion
 * Registers a jasmine reporter (for use with Mocha) that takes a screenshot of the browser when a test fails (and
```

Reply:
Ah thanks for the catch! Fixed the doblock in https://github.com/sourcegraph/sourcegraph/pull/7872/commits/415b9623f24c404f7e4ad57c375162154e9ed0d1